config/iam/KMS-policy-template.json

```diff
@@ -0,0 +1,31 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "VisualEditor0",
```

Comment:
can probably remove the Sid lines :)

Reply:
Yep let me remove and retest, if it's not necessary it just adds confusion